.travis/travis_before_install.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+# Install miniconda to avoid compiling scipy
+if test -e $HOME/miniconda2/bin ; then
+    echo "miniconda already installed."
+else
+    echo "Installing miniconda."
+    rm -rf $HOME/miniconda2
+    wget -c https://repo.continuum.io/miniconda/Miniconda2-4.1.11-Linux-x86_64.sh -O $HOME/download/miniconda.sh
```

Comment:
what happen if the directory download don't exist? If the cache get deleted, I think it won't be there, so I would be safer to create it with "mkdir -p $HOME/download", just in case.